rero_ils/config.py

```diff
@@ -982,7 +983,12 @@ def _(x):
 RERO_ILS_PERMALINK_RERO_URL = 'http://data.rero.ch/01-{identifier}'
 RERO_ILS_PERMALINK_BNF_URL = 'http://catalogue.bnf.fr/ark:/12148/{identifier}'
 
-#: RERO_ILS MEF specificconfigurations.
+#: RERO_ILS Git displayed hash
+RERO_ILS_APP_GIT_HASH = None
+if os.getenv('RERO_ILS_APP_GIT_HASH', False):
```

Comment:
This is useless. See https://invenio-app.readthedocs.io/en/latest/usage.html#instance-configuration for more details.